<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTerminalClass</class>
 <widget class="QMainWindow" name="QtTerminalClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTerminal</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSelect_Port">
    <property name="title">
     <string>Select Port</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSelect_Port"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSend_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSend_File">
   <property name="text">
    <string>Send File</string>
   </property>
   <property name="toolTip">
    <string>Send a file over the serial port</string>
   </property>
  </action>
  <action name="actionSelect_Port">
   <property name="text">
    <string>Select Port</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtTerminal.qrc"/>
 </resources>
 <connections/>
</ui>
